source/change-streams/tests/unified/change-streams.yml

```
@@ -4,7 +4,9 @@ schemaVersion: "1.7"
 
 runOnRequirements:
   - minServerVersion: "3.6"
-    topologies: [ replicaset, sharded-replicaset ]
+    # TODO(DRIVERS-XXXX): Run all possible tests against sharded clusters once we know the
```

```suggestion
    # TODO(DRIVERS-2323): Run all possible tests against sharded clusters once we know the
```

I assume you meant to reference [DRIVERS-2323](https://jira.mongodb.org/browse/DRIVERS-2323) here.